dataland-frontend/tests/e2e/specs/infrastructure/VerifyHeaders.ts

```diff
@@ -0,0 +1,42 @@
+import { createCompanyAndGetId } from "../../utils/CompanyUpload";
+
+describe("As a user, I want to ensure that security relevant headers are set.", () => {
+  function checkCommonHeaders(response: Cypress.Response<any>) {
+    expect(response.headers).to.have.property("feature-policy", "none");
+    expect(response.headers).to.have.property("referrer-policy", "no-referrer");
+    expect(response.headers).to.have.property("strict-transport-security", "max-age=31536000 ; includeSubDomains");
+    expect(response.headers).to.have.property("x-content-type-options", "nosniff");
+  }
+
+  it("test for frontend response", () => {
+    cy.request("GET", Cypress.config("baseUrl") + "/").then((response) => {
+      expect(response.headers).to.have.property("cache-control", "no-store");
+      expect(response.headers).to.have.property("content-security-policy", "frame-ancestors 'self'");
+      checkCommonHeaders(response);
+      expect(response.headers).to.have.property("x-frame-options", "sameorigin");
+    });
+  });
+
+  it("test for backend response", () => {
+    cy.ensureLoggedIn("data_uploader", Cypress.env("KEYCLOAK_UPLOADER_PASSWORD"));
+    createCompanyAndGetId("Dummy Company");
```

Comment:
Ich denke das müsste auch ohne vorhandene company funktionieren, oder?
Dann kriegt man vom BE eine leere Liste zurück - die Header kleben aber dennoch dran.

Reply:
War auch meine Erwartung, hat aber tatsächlich nicht funktioniert. Ich werde schauen ob man nicht etwas anderes benutzen kann. Am Ende geht es ja nur darum eine response vom Backend zu haben in der die Header drin sind.

Reply:
Ist angepasst ich nehme jetzt einfach den health-Endpunkt

Reply:
ok